spec/menu_spec.rb

```diff
@@ -0,0 +1,19 @@
+require './lib/menu'
+
+describe Menu do
+
+  let(:dish) { { :name => "Chips", :price => 2 } }
+
+  describe '#initialize' do
+    it 'has a dishes attribute that has individual dishes with a name and price' do
+      expect(subject.dishes).to_not eq nil
+    end
+  end
+  
+  describe 'view_menu' do
+
+    it 'displays menu with dishes and their price' do
+      expect(subject.view_menu).to include dish
```

Comment:
Its description clearly sets out expectations for what the test below is to achieve.